specification/monitor/data-plane/Microsoft.Insights/preview/2023-03-01-preview/examples/MetricBatchExample.json

```diff
@@ -0,0 +1,520 @@
+                    {
+                      "timeStamp": "2020-08-01T20:00:00Z",
+                      "average": 361,
+                      "minimum": 361,
+                      "maximum": 361
+                    },
+                    {
+                      "timeStamp": "2020-08-01T20:15:00Z",
+                      "average": 365.83088235294116,
+                      "minimum": 361,
+                      "maximum": 1018
+                    },
+                    {
+                      "timeStamp": "2020-08-01T20:30:00Z",
+                      "average": 361,
+                      "minimum": 361,
+                      "maximum": 361
+                    },
+                    {
+                      "timeStamp": "2020-08-01T20:45:00Z",
+                      "average": 363.4514925373134,
+                      "minimum": 361,
+                      "maximum": 1018
+                    },
+                    {
+                      "timeStamp": "2020-08-01T21:00:00Z",
+                      "average": 361,
+                      "minimum": 361,
+                      "maximum": 361
+                    },
+                    {
+                      "timeStamp": "2020-08-01T21:15:00Z",
+                      "average": 365.83088235294116,
+                      "minimum": 361,
+                      "maximum": 1018
+                    },
+                    {
+                      "timeStamp": "2020-08-01T21:30:00Z",
+                      "average": 361,
+                      "minimum": 361,
+                      "maximum": 361
+                    },
+                    {
+                      "timeStamp": "2020-08-01T21:45:00Z",
+                      "average": 363.4514925373134,
+                      "minimum": 361,
+                      "maximum": 1018
+                    },
+                    {
+                      "timeStamp": "2020-08-01T22:00:00Z",
+                      "average": 361,
+                      "minimum": 361,
+                      "maximum": 361
+                    },
+                    {
+                      "timeStamp": "2020-08-01T22:15:00Z",
+                      "average": 365.83088235294116,
+                      "minimum": 361,
+                      "maximum": 1018
+                    },
+                    {
+                      "timeStamp": "2020-08-01T22:30:00Z",
+                      "average": 361,
+                      "minimum": 361,
+                      "maximum": 361
+                    },
+                    {
+                      "timeStamp": "2020-08-01T22:45:00Z",
+                      "average": 363.4423791821561,
+                      "minimum": 361,
+                      "maximum": 1018
+                    }
+                  ]
+                }
+              ],
+              "errorCode": "Success"
+            }
+          ],
+          "namespace": "microsoft.storage/storageaccounts",
+          "resourceregion": "eastus"
```

Comment:
There should also be a "resourceId" property here that is missing

Reply:
I'd recommend trying to generate a real response from the latest version to make sure the example is a true reflection of what is actually being returned by the live API today.
Obviously you'll need to change any PII like resourceIds to be fake values

Reply:
Will be in the next commit